### PR TITLE
Fix scroll and icon splash problems

### PR DESCRIPTION
**Pull request by joaoachando**
Fix padding

### CODE REVIEW

**Comment by joaoachando** [comment]
For now padding is fixed
Later use SafeArea (like a container that resizes to avoid the status bar at the top of the screen, notch and other OS screens perturbations)

File: lib/tabViewController/TabViewController.dart [hunk 1]
```diff
@@ -37,79 +38,78 @@ class _TabViewControllerState extends State<TabViewController> {
...
+                Column(
+                    children: [
+                      //const Padding(padding: EdgeInsets.only(top: 25.0)),
+                      Stack(
+                          children: [Row(
+                            mainAxisAlignment: MainAxisAlignment.spaceAround,
+                            children:
+                            [//const Padding(padding: EdgeInsets.only(right: 30.0)),
+                              _buildLogos("HELP"),
+                              //const Padding(padding: EdgeInsets.only(right: 80.0)),
```
Comment (miguelrnf) [inline lib/tabViewController/TabViewController.dart:55]: delete

File: lib/tabViewController/TabViewController.dart [hunk 2]
```diff
@@ -37,79 +38,78 @@ class _TabViewControllerState extends State<TabViewController> {
...
+                ClipPath(child:
+                Container(color: Constants.PURPLE, height: height), clipper: BezierClipper()),
+                Column(
+                    children: [
+                      //const Padding(padding: EdgeInsets.only(top: 25.0)),
+                      Stack(
+                          children: [Row(
+                            mainAxisAlignment: MainAxisAlignment.spaceAround,
+                            children:
+                            [//const Padding(padding: EdgeInsets.only(right: 30.0)),
```
Comment (miguelrnf) [inline lib/tabViewController/TabViewController.dart:53]: delete

File: lib/tabViewController/TabViewController.dart [hunk 3]
```diff
@@ -37,79 +38,78 @@ class _TabViewControllerState extends State<TabViewController> {
...
+                              _buildLogos("HELP"),
+                              //const Padding(padding: EdgeInsets.only(right: 80.0)),
+                              loadHalfLogo(),
+                              //const Padding(padding: EdgeInsets.only(right: 70.0)),
```
Comment (miguelrnf) [inline lib/tabViewController/TabViewController.dart:57]: delete

File: lib/tabViewController/TabViewController.dart [hunk 4]
```diff
@@ -37,79 +38,78 @@ class _TabViewControllerState extends State<TabViewController> {
...
-                          const Padding(padding: EdgeInsets.only(right: 70.0)),
-                          _buildLogos("BELL")],
+        body: SafeArea(
+          child: Stack(
+              children: <Widget>[
+                ClipPath(child:
+                Container(color: Constants.PURPLE, height: height), clipper: BezierClipper()),
+                Column(
+                    children: [
+                      //const Padding(padding: EdgeInsets.only(top: 25.0)),
```
Comment (miguelrnf) [inline lib/tabViewController/TabViewController.dart:48]: delete

File: lib/tabViewController/TabViewController.dart (hunk 1 above)
Comment (joaoachando) [reply lib/tabViewController/TabViewController.dart]: Done in [0c1c603a](https://github.com/andretmarques/UniHub/pull/10/commits/0c1c603a26032b2e8b90cc385a3a5289c4272f69)

File: lib/tabViewController/TabViewController.dart (hunk 2 above)
Comment (joaoachando) [reply lib/tabViewController/TabViewController.dart]: Done in [0c1c603a](https://github.com/andretmarques/UniHub/pull/10/commits/0c1c603a26032b2e8b90cc385a3a5289c4272f69)

File: lib/tabViewController/TabViewController.dart (hunk 3 above)
Comment (joaoachando) [reply lib/tabViewController/TabViewController.dart]: Done in [0c1c603a](https://github.com/andretmarques/UniHub/pull/10/commits/0c1c603a26032b2e8b90cc385a3a5289c4272f69)

File: lib/tabViewController/TabViewController.dart (hunk 4 above)
Comment (joaoachando) [reply lib/tabViewController/TabViewController.dart]: Done in [0c1c603a](https://github.com/andretmarques/UniHub/pull/10/commits/0c1c603a26032b2e8b90cc385a3a5289c4272f69)

**Review by miguelrnf — APPROVED** [review]

**Pull request merged**